lust/lib/widgets/capacityInfo.dart

```diff
@@ -1,37 +1,68 @@
 import 'package:flutter/material.dart';
+import 'package:lust/models/capacity/trend.dart';
 
 class CapacityInfo extends StatefulWidget {
+
   _CapacityInfoState createState() => _CapacityInfoState();
 }
 
 class _CapacityInfoState extends State<CapacityInfo> {
+  int dummyData = 65;
+  String dummyTime = "8:00 - 23:00";
```

Comment:
Just an idea: create a function like "getOpeningHours" which return the time and hardcode the time in that new function.
The advantage: We are closer to the "real implementation".

Reply:
Very good idea. Took this approach for all the data that can change.